enterprise/api/v1/urls.py

```diff
@@ -9,7 +9,7 @@
 from enterprise.api.v1 import views
 
 router = DefaultRouter()  # pylint: disable=invalid-name
-router.register("enterprise-catalogs", views.EnterpriseCustomerCatalogViewSet, 'enterprise-catalogs')
+router.register("enterprise_catalogs", views.EnterpriseCustomerCatalogViewSet, 'enterprise-catalogs')
```

Comment:
Any reason to leave the first parameter as `"enterprise_catalogs"`?

Reply:
Yes, we want to use underscores in our API endpoint paths.

Reply:
:+1: